lib/ansible/modules/net_tools/nios/nios_host_record.py

```diff
@@ -52,9 +62,20 @@
         required: true
         aliases:
           - address
+      configure_for_dhcp:
```

Comment:
What is the reason for alias. How about renaming this option itself to `dhcp`

Reply:
I have kept the same name that we send in the json request body to infoblox client, as other variables are named and kept as same.